Sources/SwiftCurrent/Protocols/FlowRepresentable.swift

```diff
@@ -100,6 +100,16 @@ extension FlowRepresentable {
     // swiftlint:disable:next missing_docs
     public var _workflowUnderlyingInstance: Any { self }
 
+    // False positive: Inherited docs
+    // swiftlint:disable:next missing_docs
+    public static var flowRepresentableName: String { String(describing: Self.self) }
+
+    // False positive: Inherited docs
+    // swiftlint:disable:next missing_docs
+    public static func createMetadata() -> FlowRepresentableMetadata {
```

Comment:
You may want to match conventions from our `_factory` and call this metadataFactory() or createMetadataFromFactory() or something.